design-document.md

```diff
@@ -106,6 +123,132 @@ to achieve these goals:
    function output itself. {{questions-and-params}} discusses the leakage profiles of 
    various aggregation functions in more detail.
 
+### Threat model
+
+In this section, we enumerate the actors participating in the Prio system and
+enumerate their assets (secrets that are either inherently valuable or which
+confer some capability that enables further attack on the system), the
+capabilities that a malicious or compromised actor has, and potential
+mitigations for attacks enabled by those capabilities.
+
+#### Client
+
+##### Assets
+
+1. Unshared data. Clients are the only actor that can ever see the original
+   data.
+1. Unencrypted data shares.
+1. Client identity (optionally).
+
+##### Capabilities
+
+1. Individual clients can trivially and undetectably defeat the privacy property
+   by leaking data outside of the Prio system. Accordingly, such attacks are
```

Comment:
I'm making a distinction between the user and the client here (I think my use of "individual" is confusing). I'm imagining a scenario where an application vendor might adopt Prio to elicit trust from users, but ship client software that covertly leaks user data over some non-Prio channel. 

I'm not sure we can do anything about it, unless the model is extended to include a trusted OS that a client runs in, which could attest to various properties of the client (e.g., measurements of reproducible builds, or attest to a log of the client's network activity so that external observers can verify that there are no unexpected connections to `secret-cleartext-metrics.appvendor.com`).

I'm going to reword this to explicitly separate the case of an individual user/client disclosing their own data (which as you say does not compromise anyone else's privacy) and the case of a client used by many users leaking data (which is out of scope in the current model).